lib/inspec/runner.rb

```diff
@@ -101,6 +101,31 @@ def run(with = nil)
       run_tests(with)
     end
 
+    def render_output(run_data)
```

Comment:
It's kind of a bummer that some of this is a subset of the logic already in the CLI code. Can we move this out into a helper somewhere, either here in the Runner or elsewhere, so that both the CLI and the Runner can use it without needing to dual-define it? It seems like TK could benefit from the additional logic already available in the CLI code.